client/src/components/new-request/grey-box.vue

```diff
@@ -89,7 +89,8 @@
             <template v-else-if="option.type === 'assumed_name'">
               <v-col :id="option.type + '-button-checkbox-col'"
                      class="grey-box-checkbox-button text-center"
-                     v-if="entity_type_cd !== 'XLP' && entity_type_cd !== 'XLL'">
+                     v-if="entity_type_cd !== 'XLP' && entity_type_cd !== 'XLL'
+                     && entity_type_cd !== 'XCP' && entity_type_cd !== 'XSO'">
```

Comment:
As this is now quite a bit of code in the template, I recommend you use a helper function here instead.